package.json

```diff
@@ -27,7 +28,7 @@
     "redux": "^3.7.2",
     "redux-logger": "^3.0.6",
     "redux-thunk": "^2.2.0",
-    "underscore": "^1.8.3"
+    "ua-parser-js": "^0.7.23"
```

Comment:
Unless we are depending on this directly, this line should be removed.

Reply:
@gregmarr we are not depending on it directly, my thought was if I revert the change from here - and leave it in the package-lock that if someone deletes the package-lock and recreates it via install then it's not guaranteed we'll get the latest version.

Is that acceptable?

Reply:
I guess we've already made this assumption in the past, that the build machine is not going to randomly delete the package-lock file.

I've removed the direct dep.

Reply:
The build machine should be using `npm ci` and should definitely not be deleting `package-lock.json`.  However, even if you did, you should definitely get the latest version that meets the dependency requirements.